Automattic/sensei
# Style fixes in Sensei Home

Fixes #5985
Fixes #5968

### Changes proposed in this Pull Request

* It fixes the space between the sections.
* It fixes the style for the "Installed" button in mobile versions (it was with the wrong color and not centered).
* It ensures that the boxes will have the same height visually (the white part) when the section in the side has more content.
* Set a max-width of `1244px` to Sensei Home content.
* Add logo instead of "Sensei Home" title.

### Testing instructions


* Navigate to the Sensei Home, and make sure the styles related to the mentioned fixes look correct.


### Screenshot / Video

<img width="1488" alt="Screen Shot 2022-10-20 at 18 31 31" src="https://user-images.githubusercontent.com/876340/197062876-8d905b5f-61e3-4a43-901f-6f6e98b728ce.png">

<img width="503" alt="Screen Shot 2022-10-20 at 18 31 53" src="https://user-images.githubusercontent.com/876340/197062919-13272b71-6def-4344-82a4-8da4b256a982.png">
